web/modules/custom/dbcdk_community_content/src/FieldNormalizer/ImageItemFieldNormalizer.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+/**
+ * @file
+ * Contains ImageItemFieldNormalizer.
+ */
+
+namespace Drupal\dbcdk_community_content\FieldNormalizer;
+
+use Drupal\Core\Field\FieldItemBase;
+use Drupal\file\Entity\File;
+use Drupal\image\Entity\ImageStyle;
+
+/**
+ * Normalize the output for an Image field.
+ */
+class ImageItemFieldNormalizer implements FieldNormalizerInterface {
+
+  /**
+   * {@inheritdoc}
+   */
+  public function normalize(FieldItemBase $field) {
+    // Load the file by target_id.
+    /* @var \Drupal\file\Entity\File $image */
+    $image = File::load($field->get('target_id')->getString());
+
+    // Set field values to the output array.
+    $output['alt'] = $field->get('alt')->getString();
+    $output['title'] = $field->get('title')->getString();
+    $output['original'] = $image->toUrl()->getUri();
+
+    // Add urls to a processed version of the image.
+    $image_styles = [
+      'thumbnail',
+      'small',
+      'medium',
+      'large',
+    ];
+    foreach ($image_styles as $image_style_name) {
+      /* @var \Drupal\image\Entity\ImageStyle $image_style */
+      $image_style = ImageStyle::load($image_style_name);
```

Comment:
As above - but with this I only see that injecting an `EntityStorageInterface` for ImageStyles is an option.